ItemCard.tsx

```diff
@@ -0,0 +1,42 @@
+import { View, Image, Text, TouchableOpacity, StyleSheet } from 'react-native';
+
+export interface ImageData {
+    id: number;
+    url: string;
+    width: number;
+    height: number;
+}
+
+interface ItemCardProps extends ImageData {
+    isSelected: boolean;
+}
+
+export const ItemCard = ({ url }: ImageData) => {
```

Comment:
Glad to see the use of smaller components. This is always the goal when writing React/React Native.